widgets/sw.js

```diff
@@ -1,3 +1,14 @@
+const defaultActionVerb = 'inc';
+const defaultTemplate = { "type": "AdaptiveCard", "body": [{ "type": "TextBlock", "text": "You have clicked the button ${count} times" }], "actions": [{ "type": "Action.Execute", "title": "Increment", "verb": `${defaultActionVerb}`, "style": "positive" }], "$schema": "http://adaptivecards.io/schemas/adaptive-card.json", "version": "1.5" };
```

Comment:
Can this be formatted into multiple lines?

Reply:
Done.